workflow-templates/job.push.yml

```
@@ -0,0 +1,68 @@
+name: Update Image Tag in Config Repository
+
+on:
+  workflow_call:
+    inputs:
+      image_tag:
+        required: true
+        type: string
+      aws_region:
+        required: true
+        type: string
+      environment:
+        required: true
+        type: string
+      service:
+        required: true
+        type: string
+      namespace:
+        required: false
+        type: string
+        default: "liquid"
+      config_repo:
+        required: false
+        type: string
+        default: "gainsway/cluster-workloads"
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+    env:
+      values_path: "environments/liquid/${{ inputs.aws_region }}/${{ inputs.environment }}/values.yaml"
+    permissions:
+      contents: read
```

As we need to perform `git push` permissions must be write
```suggestion
      contents: write
```